x-pack/metricbeat/module/aws/cloudwatch/_meta/docs.asciidoc

```diff
@@ -22,3 +22,26 @@ The format of each resource type is service[:resourceType].
 For example, specifying a resource type of ec2 returns all Amazon EC2 resources
 (which includes EC2 instances). Specifying a resource type of ec2:instance returns
 only EC2 instances.
+
+[float]
+=== Configuration example
+[source,yaml]
+----
+- module: aws
+  period: 300s
+  metricsets:
+    - cloudwatch
+  access_key_id: '${AWS_ACCESS_KEY_ID:""}'
+  secret_access_key: '${AWS_SECRET_ACCESS_KEY:""}'
+  session_token: '${AWS_SESSION_TOKEN:""}'
+  default_region: '${AWS_REGION:us-west-1}'
+  cloudwatch_metrics:
+    - namespace: AWS/EC2
+      metricname: CPUUtilization
+      dimensions:
+        - name: InstanceId
+          value: i-0686946e22cf9494a
+    - namespace: AWS/EBS
+    - namespace: AWS/ELB
+      tags.resource_type_filter: elasticloadbalancing
+----
```

Comment:
It'd be nice to explain what these examples do, at least in the cloudwatch metricset as is more complex. It could be done using annotations in a similar fashion to what we do [here](https://www.elastic.co/guide/en/beats/metricbeat/7.2/metricbeat-metricset-jolokia-jmx.html) or [here](https://www.elastic.co/guide/en/beats/metricbeat/7.2/metricbeat-metricset-system-socket.html).

Also for this cloudwatch case we may also want to have a couple of explained examples, so users can more easily create their own configurations.

Reply:
Sounds good and thanks for pointing me to the examples!

Reply:
Thanks for adding more examples and with more details!